services/movie_session.py

```diff
@@ -0,0 +1,57 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int,
+) -> MovieSession:
+    new_movie_session = MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall=CinemaHall.objects.get(id=cinema_hall_id),
+        movie=Movie.objects.get(id=movie_id)
+    )
+    return new_movie_session
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    if session_date:
+        target_time = datetime.strptime(session_date, "%Y-%m-%d").date()
```

Comment:
Can you refactor this to use only one return?
Also you may pass string as is without conversion - django will take care of everything under the hood